BuildResidentialHPXML/tests/build_residential_hpxml_test.rb

```diff
@@ -207,16 +212,22 @@ def _check_hpxmls(workflow_dir, built_dir, test_dir, hpxml_path)
       hpxml.building_construction.conditioned_building_volume = nil
       hpxml.building_construction.average_ceiling_height = nil # Comparing conditioned volume instead
       hpxml.air_infiltration_measurements[0].infiltration_volume = nil
-      hpxml.attics.clear()
-      hpxml.foundations.clear()
-      hpxml.rim_joists.clear() # TODO
-      hpxml.refrigerators.each do |refrigerator|
-        refrigerator.adjusted_annual_kwh = nil
-      end
+      hpxml.foundations.clear
+      hpxml.attics.clear
       hpxml.foundation_walls.each do |foundation_wall|
         next if foundation_wall.insulation_assembly_r_value.nil?
         foundation_wall.insulation_assembly_r_value = foundation_wall.insulation_assembly_r_value.round(2)
       end
+      if hpxml.rim_joists.length > 0
+        (0...hpxml.rim_joists.length).to_a.reverse.each do |i|
+          next unless [HPXML::LocationLivingSpace].include? hpxml.rim_joists[i].interior_adjacent_to
+
+          hpxml.rim_joists.delete_at(i)
```

Comment:
Why are we having to do this?

Reply:
To exclude any rim joists between living space and outdoors (i.e., we decided not to handle 2nd story rim joists with the build measure).